src/main/java/Duke.java

```diff
@@ -1,17 +1,41 @@
+// import libraries here
+
+import java.util.Scanner;
+
 public class Duke {
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
-        String greetings = " Hello! I'm Duke\n" + " What can I do for you?";
-        System.out.println("____________________________________________________________");
-        String exits = " Bye. Hope to see you again soon!";
-        System.out.println(greetings);
-        System.out.println("____________________________________________________________");
-        System.out.println(exits);
-        System.out.println("____________________________________________________________");
+        greeting();
+        echo();
     }
-}
+
+    private static void echo() {
+        String line;
```

Comment:
The meaning of “line” is a bit unclear, maybe consider change it into "input"

Reply:
I will reconsider that.